test/test_pyfftw_numpy_interface.py

```diff
@@ -556,6 +557,51 @@ def test_input_maintained(self):
                 self.assertTrue(
                         numpy.alltrue(input_array == orig_input_array))
 
+    def test_on_non_writeable_array_issue_92(self):
+        '''Test to make sure that locked arrays work.
+
+        Regression test for issue 92.
+        '''
+        def copy_with_writeable(array_to_copy):
+            array_copy = array_to_copy.copy()
+            array_copy.flags.writeable = array_to_copy.flags.writeable
+            return array_copy
+
+        dtype_tuple = self.io_dtypes[functions[self.func]]
+
+        def array_type(test_shape, dtype):
+            a = dtype_tuple[1](test_shape, dtype)
+            a.flags.writeable = False
+            return a
+
+        for dtype in dtype_tuple[0]:
+            for test_shape, s, kwargs in self.test_data:
+                s = None
+
+                self.validate(array_type,
+                              test_shape, dtype, s, kwargs,
+                              copy_func=copy_with_writeable)
+
+    def test_regression_issue_92(self):
+        '''Quick regression test for issue 92.
+
+        The previous unit tests do not cover this when quick_tests is
+        run because one needs the complex dtype for failure.
+        '''
```

Comment:
Ah, sorry.  This was the test I had to implement before `validate` was fixed.  It should be removed and left just for the assertion.